src/os/ui/ol/interaction/dragboxinteraction.js

```diff
@@ -156,5 +196,5 @@ os.ui.ol.interaction.DragBox.prototype.cleanup = function() {
  * @inheritDoc
  */
 os.ui.ol.interaction.DragBox.prototype.getResultString = function() {
-  return this.extent ? this.extent.toString() : 'none';
+  return 'TODO: FIX';
```

Comment:
Does this need to be To-Done as part of this ticket and/or is there a follow-on ticket?

Reply:
Yep, forgot about this. It's actually only called to log the result, but I still need to update it.